poetry.lock

```diff
@@ -688,6 +688,14 @@ category = "main"
 optional = false
 python-versions = ">=3.6,<4"
 
+[[package]]
```

Comment:
This is not the right way to add a new dependency, the easiest way is to use `poetry add dependency_name`, which will update both pyproject.toml and poetry.lock for you.

Reply:
I have done this way now